core/vm/contracts.go

```diff
@@ -629,9 +631,10 @@ func (c *getValidator) RequiredGas(input []byte) uint64 {
 	return params.GetValidatorGas
 }
 
+// Return the validators that are required to sign this current, not yet sealed, block. If this block is
```

Comment:
Consider:
`Return the validators that are required to sign the current block.` or 
`Return the validators that are required to sign the current, possibly unsealed, block.`

Whether or not the block has been sealed yet depends on what role you're playing and when you receive the block. For example, for full nodes syncing the chain and validating blocks, the block will of course have a seal on it when doing the tx validation.

Reply:
that is very true